subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -167,20 +205,21 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
             ).build().output.contains("Build scan written to")
 
         where:
-        version << SUPPORTED
+        version << SUPPORTED_WITH_GRADLE_8_CONFIGURATION_CACHE
     }
 
-    @Requires(IntegTestPreconditions.NotConfigCached)
-    def "can use plugin #version"() {
+    // TODO: remove @IgnoreIf once 3.15 is added to the supported list, as this becomes a no-op
+    @IgnoreIf({ SUPPORTED_WITH_GRADLE_8_CONFIGURATION_CACHE.every { !it.startsWith("3.15") } })
```

Comment:
❓ I don't understand why you need the `IgnoreIf` here. Is it because the where clause is empty and then Spock fails?

Reply:
Yes, Spock fails with something like `SpockExecutionException: Data provider has no data`.